physicalplan/plan.go

```diff
@@ -122,6 +134,22 @@ func newOperator(expr parser.Expr, storage storage.Queryable, mint time.Time, ma
 	}
 }
 
+func aggregateValidation(node parser.Node, want parser.ValueType) error {
+	var t parser.ValueType
+	switch n := node.(type) {
+	case *parser.NumberLiteral:
+		t = parser.ValueTypeScalar
+	case *parser.StringLiteral:
+		t = parser.ValueTypeString
+	default:
+		return errors.New(fmt.Sprintf("unknown node type: %T", n))
```

Comment:
I am not sure if this is accurate. Those node types are known, just not supported param types, right?
We can use the same error message at line 148